CUDADataFormats/HcalDigi/src/classes_def.xml

```diff
@@ -2,51 +2,35 @@
     <class name="hcal::DigiCollectionBase<calo::common::VecStoragePolicy<std::allocator>>" />
     <class name="hcal::DigiCollectionBase<calo::common::VecStoragePolicy<calo::common::CUDAHostAllocatorAlias>>" />
 
-    <class name="hcal::DigiCollection<hcal::Flavor01, calo::common::VecStoragePolicy<std::allocator>>" />
-    <class name="hcal::DigiCollection<hcal::Flavor2, calo::common::VecStoragePolicy<std::allocator>>" />
```

Comment:
@mariadalfonso should `Flavor2` and `Flavor4` also be drmoved from CUDADataFormats/HcalDigi/interface/DigiCollection.h ?

Reply:
correct, they can be removed as well 

https://github.com/cms-sw/cmssw/pull/31720/files#diff-37c00e58e6708c06d66be199b3574bb7d2bfd6307d919fdd941eb29102655d1bR23

https://github.com/cms-sw/cmssw/pull/31720/files#diff-37c00e58e6708c06d66be199b3574bb7d2bfd6307d919fdd941eb29102655d1bR45

will you do , or shall I ?